Ginger/Ginger/ALM/ALMConnectionPage.xaml.cs

```diff
@@ -74,15 +74,15 @@ public ALMConnectionPage(ALMIntegration.eALMConnectType almConnectStyle, bool is
                 }
             }
 
-            if (!WorkSpace.Instance.BetaFeatures.Octane)
-            {
-                OctaneRadioButton.Visibility = Visibility.Hidden;
-                xDefualtImageOctane.Visibility = Visibility.Hidden;
-                if (CurrentAlmConfigurations.AlmType == GingerCoreNET.ALMLib.ALMIntegration.eALMType.Octane)
-                {
-                    CurrentAlmConfigurations.AlmType = GingerCoreNET.ALMLib.ALMIntegration.eALMType.QC;
-                }
-            }
+            //if (!WorkSpace.Instance.BetaFeatures.Octane)
+            //{
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6352807)